### PR TITLE
Vtcca 2731 file detection

### PR DESCRIPTION
**Pull request by reddy-yattapu**

### CODE REVIEW

File: app/controllers/propertyLinking/FileUploadController.scala [hunk 1]
```diff
@@ -36,19 +33,25 @@ import uk.gov.hmrc.play.bootstrap.controller.BaseController
...
-            initiateAttachmentResult <- businessRatesAttachmentService.initiateAttachmentUpload(request.body.as[InitiateAttachmentRequest].copy(destination = Some(PropertyLinkEvidence.destination)))(request, hc)
+    val initiateAttachmentRequest = request.body.as[InitiateAttachmentRequest].copy(destination = Some(PropertyLinkEvidence.destination))
+
+    (for {
+            initiateAttachmentResult <- businessRatesAttachmentService.initiateAttachmentUpload(initiateAttachmentRequest)(request, hc)
           } yield
             Ok(Json.toJson(initiateAttachmentResult))
         ).recover {
           case fileAttachmentFailed: FileAttachmentFailed =>
+            Logger.warn(Json.prettyPrint(request.body))
```
Comment (DanielConnelly) [inline app/controllers/propertyLinking/FileUploadController.scala]: please remove the logging lines.

**Comment by habeeb01** [comment]
LGTM 👍

File: app/controllers/propertyLinking/FileUploadController.scala (hunk 1 above)
Comment (reddy-yattapu) [reply app/controllers/propertyLinking/FileUploadController.scala]: Implemented

File: app/assets/javascripts/src/fileUpload.js [hunk 2]
```diff
@@ -18,11 +18,54 @@
...
+                            break;
+
+                            case "doc":
+                            mime = "application/msword";
+                            break;
+
+                            case "jpg":
+                            mime = "image/jpeg";
+                            break;
+
```
Comment (DanielConnelly) [inline app/assets/javascripts/src/fileUpload.js]: add png. image/png

Comment (reddy-yattapu) [reply app/assets/javascripts/src/fileUpload.js]: I have included PNG and also messages file updated Allowed File Types

File: conf/messages [hunk 3]
```diff
@@ -1166,10 +1166,10 @@ remove.file=Remove file
 error.summary.title=There’s been a problem
 error.summary.text=Check the following
 error.businessRatesAttachment.unavailable=Cannot upload documents - please try again later.
-error.upscan.unavailable=Cannot upload documents - please make sure only Word, Excel, PDF and JPEG files can be accepted and Maximum file size is 10MB, please try again later.
+error.upscan.unavailable=Cannot upload documents - please make sure only Word, Excel, PDF, PNG and JPEG files can be accepted and Maximum file size is 10MB, please try again later.
```
Comment (DanielConnelly) [inline conf/messages]: confirm with Mark if we are exposing we accept PNG yet.

**Comment by DanielConnelly** [comment]
LGTM 👍

**Pull request merged**